Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN . /build.env \
  && cd /vpp \
  && git pull \
  && git checkout ${VPP_COMMIT} \
- && UNATTENDED=y make install-dep bootstrap pkg-deb \
+ && UNATTENDED=y vpp_configure_args_vpp='--disable-japi --disable-vom --disable-papi' make install-dep bootstrap pkg-deb \
```

Comment:
I think `vpp_configure_args_vpp='...'` part should be used as argument to `make`, as I showed in #3. 

I just tested it locally and it seems like it's ignoring it if it's not used as argument to make.

Reply:
This didn't make any difference for me. Can you provide the exact command you've used to run `make install-dep bootstrap pkg-deb`, please?

Reply:
`make vpp_configure_args_vpp='--disable-japi --disable-vom --disable-papi' install-dep bootstrap pkg-deb`